src/monetization.js

```diff
@@ -52,10 +60,28 @@ export class Monetization extends EventEmitter {
         });
     }
 
+    /**
+     * Set session-service domain
+     * @private
+     * @param {string} domain - real URL — (**not** 'PRE' style env key)
+     * @returns {void}
+     */
+    _setSessionServiceUrl(domain) {
```

Comment:
Is this identical to the one in Identity? Easy to factor out or no?

Reply:
It is, and it's also reasonably easy (a few of these other `_setBlahUrl` functions as well), but prefer to not add that in this PR and instead do it for all of them in a separate one..